File: solutions/2798. Number of Employees Who Met the Target/2798.py
# Time complexity: O(n)
# Space complexity: O(1)
class Solution:
  def numberOfEmployeesWhoMetTarget(self, hours: List[int], 
                                    target: int) -> int:
    ans = 0

    for hour in hours:
      if hour >= target:
        ans += 1

    return ans

File: solutions/2798. Number of Employees Who Met the Target/2798.cpp
// Time complexity: O(n)
// Space complexity: O(1)
class Solution {
 public:
  int numberOfEmployeesWhoMetTarget(const vector<int>& hours, int target) {
    int ans = 0;

    for (const int hour : hours)
      if (hour >= target)
        ++ans;

    return ans;
  }
};